src/slurm_plugin/clustermgtd.py

```diff
@@ -660,6 +713,24 @@ def _handle_health_check(
                             unhealthy_node.nodeaddr,
                         )
                         self._static_nodes_in_replacement.remove(unhealthy_node.name)
+                        if self._is_protected_mode_enabled():
+                            self._handle_bootstrap_failure_nodes([unhealthy_node])
+                            log.warning(
+                                "Node bootstrap error: Node %s(%s) failed health check during replacement, "
+                                f"increase partition failure count: {self._partitions_protected_failure_count_map}, "
```

Comment:
There are still f string in the logging, let's try to use `%s`.

Reply:
Changed all to %s